urbanbroccoli/laplacian.py

```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
     return laplacian
 
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.  
+       
+       It first creates a 4-dimensional array to show all components of laplace operator at each grid point.  
+       In 4-D array,each inner 2-D array[nx,ny] represents array of all componants of laplacian Operator of a perticular grid point.  
+       Filling it according to Discetized laplace operator taking care of periodic boundary conditions.  
+       And then reshaping it into laplaction matrix .  
+       
+       Input Arguments:
```

Comment:
Remove four spaces indention just call it ``Parameters:``

Reply:
Changes done